deploy/gke.md

```diff
@@ -0,0 +1,209 @@
+
+In your project directory:
+``` 
+wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/namespace.yaml && wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/custom-types.yaml && wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/postgres.yaml && wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/rbac.yaml && wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/ingress-controller.yaml && wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/manifests/kong.yaml && wget https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/master/deploy/provider/cloud/gke-kong-proxy-loadbalancer-service.yaml
+``` 
+
+This command should create the following files:
+  - namespace.yaml : The definition of the kong's namespace
+  - custom-types.yaml : Contain Custom Types definition 
+  - postgres.yaml : Contain posgres deployment. You are free to use a DaaS as cloud-sql if needed. If you do so, this file should not be used, but you will have to configure kong.yaml and ingress-controller.yaml accordingly
+  - rbac.yaml : Contain service account / roles definition used by kong
+  - ingress-controller.yaml : The kong ingress controller deployment file. NodePort only expose port internally into node's private nerwork, so you might edit the service `kong-ingress-controller` to use the type ClusterIP, as following:
+
+```
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: kong-admin
+  namespace: kong
+spec:
+  type: ClusterIP
+  ports:
+  - name: kong-admin
+    port: 8001
+    targetPort: 8001
+    protocol: TCP
+  selector:
+    app: ingress-kong
+
+```
+  -  kong.yaml : Kong deployment file
+  - gke-kong-proxy-loadbalancer-service.yaml : The kong proxy service
+
+You can now edit theses files to fill your need (for example, you can edit theses files to use google's CloudSQL for the postgres database, or edit KONG_ADMIN_LISTEN if you wish to access kong's admin)
+If you want to use a static IP, add the IP value as `loadBalancerIP` in the service `kong-proxy` in the file gke-kong-proxy-loadbalancer-service.yaml. For example:
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: kong-proxy
+  namespace: kong
+spec:
+  externalTrafficPolicy: Local
+  type: LoadBalancer
+  ports:
+  - name: kong-proxy
+    port: 80
+    targetPort: 8000
+    protocol: TCP
+  - name: kong-proxy-ssl
+    port: 443
+    targetPort: 8443
+    protocol: TCP
+  loadBalancerIP: <my-reserved-ip-address>
+  selector:
+    app: kong
+```
+
+##### Update user rights
+
+ >Because of the way Kubernetes Engine checks permissions when you create a Role or ClusterRole, you must first create a RoleBinding that grants you all of the permissions included in the role you want to create.
```

Comment:
Remove leading " >"

Reply:
Or... I'm not sure what sort of formatting you are trying to achieve here.

Reply:
It is used for quoting Google's documentation